src/main/java/com/azure/spring/dev/tools/dependency/support/SpringProjectMetadataReader.java

```diff
@@ -40,7 +41,8 @@ public List<ProjectRelease> getProjectReleases(String projectId) {
     public String getCurrentVersion() {
         return getProjectReleases("spring-boot")
             .stream()
-            .filter(p -> p.isCurrent())
+            .filter(p -> p.getReleaseStatus().equals(ReleaseStatus.GENERAL_AVAILABILITY))
+            .filter(p -> p.getVersion().matches("2\\.\\d\\.\\d+"))
```

Comment:
Could u help provide what the response is like before and after 3.x?
What's our github action trying to do?